src/utils/mod.rs

```diff
@@ -256,16 +256,19 @@ async fn download_file_(
         (Backend::Curl, Notification::UsingCurl)
     } else {
         let tls_backend = if use_rustls {
-            TlsBackend::Rustls
-        } else {
-            #[cfg(feature = "reqwest-native-tls")]
+            #[cfg(feature = "reqwest-rustls-tls")]
```

Comment:
See https://github.com/rust-lang/rustup/pull/4118.

Reply:
Acknowledged.

Reply:
@djc Wait, this PR is functionally orthogonal with #4118, so it doesn't really have to block on it; on the contrary we'll just have to work in parallel (you on #4118 and I on https://github.com/rust-lang/rustup/pull/4105#discussion_r1889971879) and call it a day?

Reply:
Yes, makes sense!